tests/Feature/InstallTest.php

```diff
@@ -69,6 +69,7 @@ public function testInstall(): void
 			'albums',
 			'base_albums',
 			'web_authn_credentials',
+			'webauthn_credentials',
```

Comment:
The first one should be removed, because it is replaced be the second. Right?

Reply:
yes.